pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -180,6 +180,40 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		},
 	}
 
+	if options.DNSConfig != nil {
+		_, dnsPortStr, err := net.SplitHostPort(options.DNSConfig.BindAddress)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse DNSConfig.BindAddress: %v", err)
+		}
+		dnsPort, _ := strconv.Atoi(dnsPortStr)
```

Comment:
@liggitt similar message as the first error or is there a better way to chain the splithostport and strconv?